apps.json

```diff
@@ -2185,6 +2185,13 @@
         "state": "notworking",
         "url": "https://github.com/abeudin/proftpd_ynh"
     },
+    "pterodactyl": {
+        "branch": "master",
+        "revision": "HEAD",
+        "maintained": true,
+        "url": "https://github.com/aeris-studio/pterodactyl_ynh",
+        "state": "inprogress"
+    }
```

Comment:
You forgot a comma after the `}` ;)

Reply:
@arcdigital's comment is quite relevant actually. Why fork the repo already hosted in YunoHost-Apps, and not contribute your improvements back into it?

Reply:
Comma added